scripts/examples/typescript/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "xen-api-ts",
```

Comment:
AFAIK, xen-api already exists in the npm repository, so it can't be used

Reply:
maybe "@xapi-project/xen-api" or `xen-api-bindings` then? `xen-api-ts` isn't great :/  and I'd avoid "@xenserver/xen-api" since people might assume it's a fully supported SDK

Reply:
Why not xen-api-sdk? We can always make it a full SDK later.

In fact, I was about to ask why is this placed under scripts/examples and not in ocaml/sdk-gen so we can package it easier in the SDK zip? I know XenAPI.py is in scripts.examples, but this is because it can be (or is) used on the server CLI - isn't the typescript module meant for client use?

Reply:
I also noticed that `xen-api-ts` is already published under Fei's profile. Given ther are vairous other flavours ov xenapi/xen-api/etc/ we may also want to use an "organisation" profile to show that this is the official module published directly by/from the upstream xapi-project repo. See for example https://pypi.org/project/XenAPI/ where we publish undnr Citrix XenServer (maybe this should also change to plain XenServer).

Reply:
>  We can always make it a full SDK later.

if so we could just add @xenserver/xen-api later and for now keep this as @xenserver/xen-api-bindings. If it's not an sdk, I don't see why we should call it that :/ 


If we use a xenserver profile and prepend our packages with @xenserver we should be ok? but we need to ask around before we start doing that, of course

Reply:
I used `xen-api-ts` just test if the github actions works, and agree to use an "organisation" profile

Reply:
> if so we could just add @xenserver/xen-api later and for now keep this as @xenserver/xen-api-bindings. If it's not an sdk, I don't see why we should call it that :/

Full SDK means language bindings, which is what this module is not 😃 

Even if it were, I see `bindings` as a term that might appear in a library's description, but not a term usually chosen for a library's title. I don't think it is a huge issue to use `xen-api-sdk` immediately, but if you really hate it, how about `xen-api-lib` or `xenserver-lib` (we call the C SDK `xenserverlib` too).

Since this is becoming the official typescript SDK/module/library, we should finalise the name as early as possible. Publishing one name now and a different name later does not make for a great user experience, it's rather confusing.

Reply:
`xen-api-lib` is OK for me.